web/pgadmin/tools/sqleditor/static/js/components/sections/MainToolBar.jsx

```diff
@@ -501,6 +503,11 @@ export function MainToolBar({containerRef, onFilterClick, onManageMacros, onAddT
         <PgButtonGroup size="small">
           <PgIconButton title={gettext('Open File')} icon={<FolderRoundedIcon />} disabled={!queryToolCtx.params.is_query_tool}
             shortcut={queryToolPref.btn_open_file} onClick={openFile} />
+          <PgIconButton title={gettext('Open in new tab')} icon={<KeyboardArrowDownIcon />} splitButton disabled={!queryToolCtx.params.is_query_tool}
```

Comment:
Rename 'Open in new tab' to 'Open in a new tab'.

Reply:
Done.